raw_data/preprocessed/africa/lesotho_english.txt

```diff
@@ -0,0 +1,379 @@
+
+Record keeping
+(93) [1] The Minister may, by notice in the Gazette, prescribe the activities for which records shall be kept for the purposes of this Act, the contents of those records and the manner in which they shall be kept. [2] The records kept in accordance with subsection [1] or any other records kept, for purposes of this Act, at the site of an establishment or undertaking shall be made available to the Director or an environment inspector for the purposes of: [a] an environmental audit; [b] environmental monitoring; [c] pollution control; [d] inspection; and [e] any other purpose that may be prescribed by the Minister. [3] The records kept pursuant to this section and any other records kept for purposes of this Act or the regulation made under this Act shall be copied and transmitted to the Director at such intervals as the Director may determine. [4] The Director shall keep all records transmitted to the Director pursuant to subsection [3] and maintain their confidentiality where circumstances so require. 
+
+PART XII - INTERNATIONAL ENVIRONMENTAL CONVENTIONS
+
+Conventions and treaties on the environment
+(94) [1] Where Lesotho is a party to an international or regional convention or agreement, concerning the management of the environment or natural resources, the Director shall, in consultation with the relevant Line Ministry: [a] initiate and prepare legislative proposals for consideration by the relevant Ministry for purposes of implementing those international or regional conventions or agreements; and [b] identify appropriate measures necessary for the implementation of conventions or agreements. [2] The Director shall keep a register of all international conventions or agreements concerning the management of the environment or natural resources to which Lesotho is a party. 
+
+PART XIII - INFORMATION, EDUCATION AND PUBLIC AWARENESS
+
+Freedom of access to environmental information
+(95) [1] A person who desires to obtain information relating to the implementation of this Act or any other information concerning the management of the environmental or natural resources shall subject to section 93[4] have access to that information. [2] A person referred to in subsection [1] shall: [a] apply to the Director in writing indicating the type of information he desires to obtain; and [b] pay a fee, which may be prescribed by the Minister. [3] The Director may grant access to the information referred to in this section, on such terms and conditions, as the Director may deem necessary. [4] Freedom of access to information pursuant to this section does not extend to proprietary information, which shall be treated by the Director as confidential. [5] In this section, “proprietary information” means information relating to any manufacturing process, trade secret, trade mark, copyright, patent or formula protected by any other law.
```

Comment:
Please replace `management of the environmental` with `management of the environment`. Also, please replace `section, on such` with `section on such`. Furthermore, please replace `conditions, as the` with `conditions as the`.